hermes/probe/probe.go

```diff
@@ -107,8 +95,9 @@ func (mp *Probe) Init(name string, opts *options.Options) error {
 // This is a required method to implement the cloudprober.Probes.Probe interface.
 // Arguments:
 //	- ctx: context provided for cancelling probe.
-//	- metricChan: unidirectional channel used for sending metrics to be surfaced.
-func (mp *Probe) Start(ctx context.Context, metricChan chan<- *cpmetrics.EventMetrics) {
+//	- metricChan: bidirectional channel used for sending metrics to be surfaced.
+//		-> Must be bidirectional to satisfy cloudprober.Probes.Probe interface.
```

Comment:
Nit: No need for the arrow ("->").

Reply:
Sorry, I'm used to using:
- XXX
     -> YYY
- ZZZ

I've changed this to use:
- XXX
     - YYY